### PR TITLE
button done

### PR DESCRIPTION
**Pull request by theVivek-kumar**
button component is done please review

### CODE REVIEW

File: button/button.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+.btn--danger1 {
+    background-color: var(--RED);
+    color: white;
+    border: 1px solid red;
+
+
+}
+
+.btn--primary1 {
+    background-color: wheat;
```
Comment (swethaogeti) [inline button/button.css:76]: keep it consistent declare all the colors at one place.

File: docs .html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                
+                          <div class=" alert alerts-Danges">
+                            <strong>Danger</strong> This is a Danger alerts.
+                           </div>
+                          </div>
+                
+                     </div>
+                     
+            
+
```
Comment (swethaogeti) [inline docs .html]: remove the extra space . rest is good!!

File: button/button.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
+@import url(/style.css);
+/*buttons css*/
+.btns {
+    display: flex;
+    justify-content: space-around;
+    margin: 20px;
```
Comment (dev-ashishsethi) [inline button/button.css:6]: use rem

File: button/button.css [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    padding: 5px;
+    background-color: rgb(255, 255, 255);
```
Comment (dev-ashishsethi) [inline button/button.css:8]: use variable for color

File: docs .html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+                        <div class=" alert alerts-info">
+                            <strong>Info</strong> This is a info alerts.
+                        </div>
+                
+                        <div class="   alert alerts-Warning">
+                            <strong>Warning</strong> This is a Warning  alerts.
+                           </div>
+                
+                          <div class=" alert alerts-Danges">
+                            <strong>Danger</strong> This is a Danger alerts.
```
Comment (dev-ashishsethi) [inline docs .html]: good that you are using semantic

**Review by dev-ashishsethi — APPROVED** [review]
do checkout comments

File: button/button.css [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+    width: 60%;
+    margin: 10px auto;
+    border: .1px solid rgb(224, 224, 224);
+}
+
+.btn {
+    background-color: orange;
+    border: none;
+    padding: .5rem;
+    border-radius: 10px;
```
Comment (swethaogeti) [inline button/button.css:20]: anything less tham 1 rem try using px for smaller values

File: button/button.css [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+/*button varient*/
+.btn--info1 {
+    background-color: orangered;
+    color: white;
+    border-radius: 27px;
+}
+
+.btn--success1 {
+    background-color: whitesmoke;
+    color: rgb(44, 44, 44);
```
Comment (swethaogeti) [inline button/button.css:59]: use var( )

**Review by swethaogeti — APPROVED** [review]

**Pull request merged**